tapis-project/tapis-ui
# task/TUI-189 - File Permissions

## Overview:

File Permissions modal

## Related Github Issues:

- [TUI-189](https://github.com/tapis-project/tapis-ui/issues/189)

## Summary of Changes:

- Added File Permissions modal
- Added FileStat component, api and hooks
- Added FileOperation component, api and hooks

## Testing Steps:

1. Try running chmod 600 on a file or a directory (recursively) and make sure it works

## UI Photos:

## Notes:

- Waiting on ability to list all users with permissions assigned to a file object before continuing (this will be another tab)
